config/rzl/utils.js

```diff
@@ -64,9 +64,16 @@ export const tasmota = {
       defaultValue: "offline"
     },
   }),
-  icon_color: function (name: string, on_color: Color = hex("#00FF00")) {
-    return (state: State) =>
-      state[`${name}_online`] == "offline" ? hex("#888888") : (state[name] == "on" ? on_color : hex("#000000"))
+  icon_color: (name: string, on_color: Color = hex("#00FF00")) => function (state: State) {
```

Comment:
Please don't mix arrow functions and anonymous functions. Please use two arrows instead.